app/src/main/java/com/stevesoltys/seedvault/settings/SettingsFragment.kt

```diff
@@ -244,10 +255,41 @@ class SettingsFragment : PreferenceFragmentCompat() {
         backupLocation.summary = storage?.name ?: getString(R.string.settings_backup_location_none)
     }
 
-    private fun setAppBackupStatusSummary(lastBackupInMillis: Long) {
-        // set time of last backup
-        val lastBackup = lastBackupInMillis.toRelativeTime(requireContext())
-        backupStatus.summary = getString(R.string.settings_backup_status_summary, lastBackup)
+    private fun setAppBackupStatusSummary(lastBackupInMillis: Long?) {
+        if (lastBackupInMillis != null) {
+            // set time of last backup
+            val lastBackup = lastBackupInMillis.toRelativeTime(requireContext())
+            backupStatus.summary = getString(R.string.settings_backup_status_summary, lastBackup)
+        }
+    }
+
+    private fun setAppBackupSchedulingSummary(workInfo: WorkInfo?) {
```

Comment:
This function is on the larger side, maybe could use some documentation?

Reply:
Alright, added.